py-rattler/rattler/virtual_package/virtual_package.py

```diff
@@ -1,11 +1,154 @@
 from __future__ import annotations
 
-from rattler.rattler import PyVirtualPackage
+from rattler.rattler import PyVirtualPackage, PyOverride, PyVirtualPackageOverrides
 from typing import List
 
 from rattler.virtual_package.generic import GenericVirtualPackage
 
 
+class Override:
+    _override: PyOverride
+
+    @classmethod
+    def _from_py_override(cls, py_override: PyOverride) -> Override:
+        """Construct Rattler Override from FFI PyOverride object."""
+        override = cls.__new__(cls)
+        override._override = py_override
+        return override
+
+    @classmethod
+    def default_env_var(cls) -> Override:
+        """
+        Returns the default environment variable override.
```

Comment:
This is a bit confusing. Maybe we can write it as:

```suggestion
        Returns a new instance to indicate that the default environment variable should overwrite the detected information from the host if specified.
```

And something similar for the other functions.